src/adapters/github.js

```diff
@@ -315,21 +315,23 @@ class GitHub extends PjaxAdapter {
 
     $.ajax(cfg)
       .done((data, textStatus, jqXHR) => {
```

Comment:
Why to use anonymous execution ? This doesn't work ?

```suggestion
      .done(async (data, textStatus, jqXHR) => {
```

Reply:
Yep this does not work

Reply:
I wonder why it doesn't work?

Reply:
it did not work when i tested it. I also could not find any relevant info on google. So my coclusion is it doesn't work